src/ownership-and-deconstruction.rst

```diff
@@ -256,71 +256,55 @@ Variable ``x`` is captured using a unique immutable borrow.
 
 .. _fls_77scxuomlbgs:
 
-Passing Conventions
--------------------
+Value Passing
+-------------
 
 .. rubric:: Legality Rules
 
-:dp:`fls_fvwx2ufeyzcs`
-A :t:`passing convention` is a mechanism by which a :t:`value` is passed to and
-from a :t:`function`.
-
-:dp:`fls_1gyeqfpe7m1m`
-A :t:`value` is subject to a :t:`passing convention` when the :t:`value` is
-
-* :dp:`fls_jag0ud2lv08`
-  Assigned using an :t:`assignment expression`,
-
-* :dp:`fls_bkuz12srez4`
-  Bound to a :t:`function parameter` in a :t:`call expression`,
-
-* :dp:`fls_owltkb5i2lah`
-  Bound to a :t:`pattern`,
-
-* :dp:`fls_olqpm32j8va6`
-  Captured by a :t:`capture expression`,
-
-* :dp:`fls_czune894326w`
-  Returned from a :t:`function`.
+:dp:`fls_v4djthWlamKC`
+:t:`Passing <pass>` is the mechanism by which a :t:`value` is transferred between
+:t:`[place]s`. **place is not the appropriate term here**
 
-* :dp:`fls_qi148dixkp0w`
-  **More?**
+:dp:`fls_fvwx2ufeyzcs`
+A :t:`passing convention` is the mechanism that defines how a :t:`value` is
+transferred between :t:`[place]s`.
 
 :dp:`fls_h2pgsij1rbms`
-A :t:`by copy type` is a :t:`type` that implements the :std:`core::marker::Copy`
+A :t:`copy type` is a :t:`type` that implements the :std:`core::marker::Copy`
 :t:`trait`.
 
 :dp:`fls_yx2knbby70fy`
-A :t:`value` of a :t:`by copy type` is passed :dt:`by copy`. Passing :t:`by
+A :t:`value` of a :t:`copy type` is :t:`[pass]ed` :dt:`by copy`. Passing :t:`by
 copy` does not change the :t:`owner` of the :t:`value`.
 
 :dp:`fls_6ul3f6v0foma`
-A :t:`by move type` is a :t:`type` that does not implement the
-:std:`core::marker::Copy` :t:`trait`.
+A :t:`move type` is a :t:`type` that implements the :std:`core::marker::Sized`
+:t:`trait` and that is not a :t:`copy type`.
 
 :dp:`fls_3ztdz02efeoc`
-A :t:`value` of a :t:`by move type` is passed :dt:`by move`. Passing :t:`by
+A :t:`value` of a :t:`move type` is :t:`[pass]ed` :dt:`by move`. Passing :t:`by
 move` changes the :t:`owner` of the :t:`value`.
 
-:dp:`fls_ljc1k8ms16gp`
-Passing :dt:`by value` is either passing :t:`by copy` or passing :t:`by move`.
+:dp:`fls_konzgoybhfqm`
+A :t:`value` of a :t:`place expression` shall be :t:`[pass]ed` :t:`by move` only
```

Comment:
This sounds like a fact (tool guaranteed) rather than a rule (programmer requirement).

Reply:
After discussing this live, it is in fact a rule because the programmer can control the passing convention explicitly.